libbeat/dashboards/import_dashboards.go

```diff
@@ -639,49 +645,77 @@ func (imp Importer) ImportArchive() error {
 	return nil
 }
 
+func (imp Importer) subdirExists(parent string, child string) bool {
+	if _, err := os.Stat(path.Join(parent, child)); err != nil {
+		return false
+	}
+	return true
+}
+
 // import Kibana dashboards and index-pattern or only one of these
 func (imp Importer) ImportKibana(dir string) error {
 
 	var err error
 
+	if _, err := os.Stat(dir); err != nil {
+		return fmt.Errorf("No directory %s", dir)
+	}
+
+	check := []string{}
 	if !imp.cl.opt.OnlyDashboards {
-		err = imp.ImportDir("index-pattern", dir)
-		if err != nil {
-			return fmt.Errorf("fail to import index-pattern: %v", err)
-		}
+		check = append(check, "index-pattern")
 	}
 	if !imp.cl.opt.OnlyIndex {
-		err = imp.ImportDir("dashboard", dir)
+		check = append(check, "dashboard")
+	}
+
+	types := []string{}
+	for _, c := range check {
+		if imp.subdirExists(dir, c) {
+			types = append(types, c)
+		}
+	}
+
+	if len(types) == 0 {
```

Comment:
In case the script is run for Filebeat with the `-only-dashboards` argument, it shouldn't return an error as there aren't dashboards to import.